### DISCUSSION TITLE
Question about zero inflated negative binomial describe_posterior output

### DISCUSSION THREAD

**Discussion by StromTroopers** [easystats/bayestestR#492] [Show and tell]
Hello, everyone,  would like to ask some questions about the **describe_posterior** function that you have developed in the bayestR package.

First of all, I would like to say that it is a package that I appreciate very much and whose documentation is of high quality, thank you for that.

My question is about using this function to check the output of the following model:

```R
model = brm(Nb_species ~  environment + time  , data = tab,
                     family = zero_inflated_negbinomial(),
                     iter  = 10000, chains = 2,
                     seed  = 1234,
                     cores = 1,thin=5,backend = "rstan")
```

In this model, I have two explanatory variables for **Nb_species**, one continuous quantitative (`time`)  (time in millions of years), and the other qualitative  :  (`environment`) with three modalities (**sky**, **sea**, **land**, with sky as reference)

I then used the package and got :

```R
describe_posterior(model,ci = 0.89)
#> Summary of Posterior Distribution
#> 
#> Parameter   |   Median  |         89% CI |     pd |          ROPE | % in ROPE |  Rhat   |     ESS
#> -------------------------------------------------------------------------------------------------
#> b_Intercept |    -0.86  | [-1.24, -0.48] |   100% | [-0.10, 0.10] |        0% | 0.999.  | 2015.00
#> b_sea       |     0.80  | [ 0.39,  1.14] |   100% | [-0.10, 0.10] |        0% | 1.000.  | 1723.00
#> b_land      |    -0.08  | [-0.68,  0.50] | 57.95% | [-0.10, 0.10] |    20.04% | 0.999.  | 2043.00
#> b_time      | 7.69e-03  | [ 0.00,  0.01] | 99.75% | [-0.10, 0.10] |      100% | 1.000.  | 2144.00 
```

I would like to know if the following conclusions I'm doing in regard to the summary output are right (I'm not sure about the exponential) :

- There is an effect of the sea modality because the interval [ exp (0.39), exp(1.14) ] is well outside of 1. There is thus in median exp(0.80) = **2.22 times** more species in the **sea** environment compared to the **sky** environment.
(or do we have to say : there is in median exp(-0.86+0.80) = 0.94 time less **species** in the sea compared to **sky**? )e

- There is no effect of the land modality because the interval [ exp (-0.68), exp(0.50) ] contains 1. There is thus the same number of species in **land** environment compared to the **sky** environment.

- The variable **time** has a coefficient of 0.00769. This means that for each increase of one million years, the expected logarithmic number of species found increases by 0.00769.

Thank you very much for your help and time, it would mean a lot to me to have explanation on this.

**Comment by mattansb** [comment]
> There is an effect of the sea modality because the interval [ exp (0.39), exp(1.14) ] is well outside of 1. There is thus in median exp(0.80) = 2.22 times more species in the sea environment compared to the sky environment.

Sounds right to me.

> There is no effect of the land modality because the interval [ exp (-0.68), exp(0.50) ] contains 1. There is thus the same number of species in land environment compared to the sky environment.

Not quite. The results here aren't clear cut - the null was not rejected, but (using the default ROPE) it also is not strongly supported.

> The variable time has a coefficient of 0.00769. This means that for each increase of one million years, the expected logarithmic number of species found increases by 0.00769.

I would say that for every million years, the number of species increases by a factor exp(0.00769)=1.00772.

**Reply by StromTroopers** [reply]
Thanks a lot for your reply. 
I just add another question regarding the %rope definition. 
Let's say I have a distribution of the coefficients such as that : 

<img width="1256" alt="Capture d’écran 2022-03-19 à 12 36 16" src="https://user-images.githubusercontent.com/38229558/159119542-7735661c-4acd-4ecf-9610-9fc22bcdc60e.png">



I wrote two cases (different from the table in my first post), because I don't know what the right way to define a string is.

In **case1**, I would say that the sea modality has 0.28% of values different from 1. 
So I can conclude that it is significantly different from 1, but it is hard for me to conclude that it has significantly fewer species than the sky (the intercept). But if I take as in **case2** as a %rope every value from 1.1 to 0, then here I get directly the number of sea coef values supporting a conclusion of less number of species, right? Isn't it better if I want to conclude that I have overall 98.5%  coefficient values showing less number of species compared to the sky? 

Thanks a lot for your help.
